script/hassfest/mypy_config.py

```diff
@@ -11,25 +11,6 @@
 
 from .model import Config, Integration
 
-# Modules which have type hints which known to be broken.
-# If you are an author of component listed here, please fix these errors and
-# remove your component from this list to enable type checks.
-# Do your best to not add anything new here.
-IGNORED_MODULES: Final[list[str]] = [
```

Comment:
I would suggest to only remove `sonos` from the list and to leave the logic intact.